puppet/modules/jenkins_node/manifests/unittests.pp

```diff
@@ -164,11 +164,6 @@
   # Databases
   include jenkins_node::postgresql
 
-  # RVM
-  if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] != '9' {
-    include jenkins_node::rvm
-  }
-
   # rbenv
   if $facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] != '7' {
```

Comment:
Unrelated, but this can be simplified too now. But I see that approach can be taken wider so let's not include it here.

Reply:
I think we should do a sweeping cleanup once we migrated the CI nodes to EL9, then we can remove all el7 and 8 specialcases.